explainerdashboard/to_html.py

```diff
@@ -281,14 +219,3 @@ def input(feature:str, value, disabled:bool=False)->str:
     <input id="{feature}" type="text" value="{value}" name="{feature}" {'disabled' if disabled else ''}>
 </div>
     """
-
-def jumbotron(title:str, description:str)->str:
```

Comment:
against is used by ExplainerHub static html export, so need to create a replacement...

Reply:
What is the preferred way to test the layout/s? Could you point me to a good test routine?

Reply:
euhhh, not being a webdev, I think my routine when I originally developed it was

1. export to .html
2. open in browser
3. see if it looks good or if there are any errors.

(also need to include the static exports to the testing battery. In general working on updating the tests to pytest now, and need to extend the coverage)